Assets/Python/CityNameManager.py

```diff
@@ -3372,8 +3377,11 @@ def onRevolution(iCiv):
 	"Budapeszt"		:	"Budapest",
 	"Budin"			:	"Budapest",
 	"Ofen"			:	"Budapest",
-	"Buju"			:	"Buju",
-	"Tieling"		:	"Buju",

```

Comment:
What happened to this?

Reply:
Tieling was an unused city name.
Buju was a Korean city name for a tile in Manchuria.
That was wrongly placed so I fixed it.
Buju is now indexed to Fushun.

Reply:
So no city is called Buju anymore?

Reply:
No, it is now a name for a different tile which is indexed as Fushun.